packages/api/src/routes/medical/__tests__/mapi/e2e/shared.ts

```diff
@@ -0,0 +1,27 @@
+import { MetriportMedicalApi } from "@metriport/api-sdk";
+import { FhirClient } from "../../../../../external/fhir/api/api";
+import { baseURL } from "../../../../__tests__/shared";
+import { getEnvVarOrFail } from "../../../../../shared/config";
+import { MedplumClient } from "@medplum/core";
+
+export const ACCOUNT_PATH = "/internal/admin/cx-account";
+export const CUSTOMER_PATH = "/internal/customer";
+export const MAPI_ACCESS = "/internal/mapi-access";
+
+export const testId = getEnvVarOrFail("TEST_ACC_ID");
+export const testApiKey = getEnvVarOrFail("TEST_API_KEY");
+export const fhirHeaders = { headers: { "x-api-key": testApiKey } };
+
+export const fhirApi = new MedplumClient({ baseUrl: baseURL });
+export const medicalApi = new MetriportMedicalApi(testApiKey, { baseAddress: baseURL });
+
+export enum ResourceType {
+  Organization = "Organization",
+  Patient = "Patient",
+  DocumentReference = "DocumentReference",
+}
```

Comment:
Why creating this enum? We can just use the literals on the calls to Medplum SDK and it recognizes the type accordingly.